sdk/core/System.ClientModel/src/Pipeline/PipelineTransport.cs

```diff
@@ -0,0 +1,83 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Threading.Tasks;
+
+namespace System.ClientModel.Primitives;
+
+// TODO: MessageClassifier will come in a later PR.
+
+public abstract class PipelineTransport : PipelinePolicy
+{
+    /// <summary>
+    /// TBD: needed for inheritdoc.
+    /// </summary>
+    /// <param name="message"></param>
+    public void Process(PipelineMessage message)
```

Comment:
we should seal the template methods.

Reply:
Will fix.

Reply:
Ah, I think I tried this in the past too and hit this: https://learn.microsoft.com/en-us/dotnet/csharp/misc/cs0238?f1url=%3FappId%3Droslyn%26k%3Dk(CS0238)